website/static/js/filepage/index.js

```diff
@@ -198,12 +199,17 @@ var FileViewPage = {
         $('.file-view-panels').removeClass().addClass('file-view-panels').addClass(fileViewPanelsLayout);
 
         m.render(document.getElementById('toggleBar'), m('.btn-toolbar[style=margin-top:20px]', [
-            ctrl.canEdit() ? m('.btn-group', {style: 'margin-left: 0;'}, [
-                m('.btn.btn-sm.btn-danger.file-delete', {onclick: $(document).trigger.bind($(document), 'fileviewpage:delete')}, 'Delete')
+            ctrl.viewUrl ? m('.btn-group', {}, [
+                m('a', {href: ctrl.viewUrl, target: '_blank', class: 'btn btn-sm btn-success'}, 'View ', [
```

Comment:
We already have a "View" button in the top-right corner, so this text should probably be more unique. Maybe "View on <addon>", e.g. "View on Dropbox"?